packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -136,33 +136,29 @@ export class SeenGossipBlockInput {
 
       if (blobKzgCommitments.length === blobsCache.size) {
         const allBlobs = getBlockInputBlobs(blobsCache);
-        resolveAvailability({...allBlobs, blobsSource: BlobsSource.gossip});
+        const blockData = {...allBlobs, blobsSource: BlobsSource.gossip, fork: cachedData.fork} as BlockInputDataBlobs;
```

Comment:
why is the type cast here?

Reply:
removed cast, but i guess it was needed downstream in peerdas, will resolve it there if still required